packages/core/src/utils/colors-vars-map.ts

```diff
@@ -224,24 +224,37 @@ export enum ContentColorByName {
   PECAN = "pecan"
 }
 
+/**
+ * @deprecated
+ */
 export enum StateSelectedColor {
-  POSITIVE = "--positive-color-selected",
-  NEGATIVE = "--negative-color-selected",
-  PRIMARY = "--primary-selected-color",
-  WARNING = "--warning-color-selected"
+  POSITIVE = "positive",
+  NEGATIVE = "negative",
+  PRIMARY = "primary",
+  WARNING = "warning"
 }
+
+/**
+ * @deprecated
+ */
 export enum StateSelectedHoverColor {
-  POSITIVE = "--positive-color-selected-hover",
-  NEGATIVE = "--negative-color-selected-hover",
-  PRIMARY = "--primary-selected-hover-color"
+  POSITIVE = "positive",
+  NEGATIVE = "negative",
+  PRIMARY = "primary"
 }
 
+/**
+ * @deprecated
+ */
 export const ElementAllowedColor = {
   ...ContentColorByName,
   ...StateSelectedColor,
   ...StateSelectedHoverColor
 };
 
+/**
+ * @deprecated
+ */
```

Comment:
Why? It's a type, I think we want it

Reply:
Since this type is using the deprecated enums, I created a new type that is exported. Should we still keep this one? I thought it would be confusing

Reply:
Oh got it, so let's keep it this way